src/mmword.c

```diff
@@ -288,8 +286,8 @@ l7220:  i=i+m-1;
              fprintf(f3_fp_, "%s\n", tmpLine);
            }
            fprintf(f3_fp_, "%s\n", line1_[i]);
-                                     /* Output original deleted lines */
-           /*freeTempAlloc();*/ /* Clear vstring stack */
+                                     // Output original deleted lines
```

Comment:
BTW, the reason for the funny indenting here IIUC is that they are *right aligned*, to 79-80 characters. I think we can just stop doing this, I much prefer putting comments about a line of code on the previous line if it doesn't fit in the margin.

Reply:
I made another commit, is this what you mean?